examples/rl/actor_critic_cartpole.py

```diff
@@ -56,6 +57,7 @@
 gamma = 0.99  # Discount factor for past rewards
 max_steps_per_episode = 10000
 env = gym.make("CartPole-v0")  # Create the environment
+# Adding `render_mode='human'` will show the attempts of the agent
```

Comment:
This comment about `render_mode='human'` applies to the line above right? Can you move it before line 59?